src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -510,23 +510,27 @@ void describeChannelBundles(ColibriConferenceIQ iq)
      */
     void describeEndpoints(ColibriConferenceIQ iq)
     {
-        synchronized (transportManagers)
+        for (Iterator<Endpoint> i = getEndpoints().iterator(); i.hasNext(); )
```

Comment:
What's the reason for an iterator instead of foreach?

Reply:
Cause copy-paste is easier :P, will fix all those